### ISSUE TITLE
Unexpected modifier sealed warning

### ISSUE THREAD

**Issue by wimdeblauwe** [openrewrite/rewrite-spring#276]
I have used openrewrite for the first time, running `org.openrewrite.java.spring.boot2.WebSecurityConfigurerAdapter`. The run shows a warning:

```
[WARNING] Unexpected modifier sealed
```
for any `sealed interface` class I have. The receipe has run fine despite the warning, but maybe something to investigate?

**Comment by timtebeek** [comment]
Hi Wim, good to see you here & glad to hear you're trying out OpenRewrite! :)
Could you let me know which version you're running?

We just had a release last week, and I see we still need to update [the documentation](https://docs.openrewrite.org/reference/recipes/java/spring/boot2/websecurityconfigureradapter). /cc @mike-solomon 

The latest versions can be run with:
```bash
mvn org.openrewrite.maven:rewrite-maven-plugin:4.39.0:run \
  -Drewrite.recipeArtifactCoordinates=org.openrewrite.recipe:rewrite-spring:4.32.0 \
  -DactiveRecipes=org.openrewrite.java.spring.boot2.WebSecurityConfigurerAdapter
```

Could you let me know if that already resolves your issues? As 4.35.0 has [relevant changes](https://github.com/openrewrite/rewrite/pull/2569).

**Comment by timtebeek** [comment]
Glad to hear that works! The message you got indicated that any files containing the `sealed` keyword could not be parsed, and would not have been updated by any recipes. It's unlikely that affected the particular migration that you ran, as I don't easily see that touching any files using `sealed`. Either way, it's been fixed when using Rewrite Maven plugin 4.39.0+. 

Mike is updating the docs right now, which you should see reflected in [openrewrite/rewrite-docs](https://github.com/openrewrite/rewrite-docs) soon, and then right after that on [docs.openrewrite.org](https://docs.openrewrite.org/) as well.

The trick about running commands as a one off without updating the pom.xml is (for now) only documented on individual recipe pages. Here's for instance the link to [the WebConfigurerAdapter recipe](https://docs.openrewrite.org/reference/recipes/java/spring/boot2/websecurityconfigureradapter) that you used. There's a Maven Command Line tab that helpfully provides the full command.
![image](https://user-images.githubusercontent.com/1027334/214090774-9552cf08-e7f6-435a-a9f1-7cf1b4b52ee0.png)

I [added that last year](https://github.com/openrewrite/rewrite-recipe-markdown-generator/pull/11) as an OSS contribution, just because I like the convenience so much as well.

As always, if you have any other questions feel free to reach out, either here or on OpenRewrite/Foojay Slack, and hope to see you at FOSDEM or another event this year! :)

**Comment by timtebeek** [comment]
And now the docs have also been updated; preventing this issue in the future. Thanks again!

And let me know if you ever need any help getting your library users to migrate to newer versions. :)
